wlm_server/task/measure.py

```diff
@@ -43,3 +43,12 @@ def pop(self) -> MeasureInfo | None:
         except IndexError:
             return None
         return item[1]
+
+    def remove(self, channel: int):
+        """Removes the measurement of the given channel from the measurement queue.
+        
+        Args:
+            channel: Target channel.
+        """
+        self._queue = list(filter(lambda item: item[1].channel != channel, self._queue))
```

Comment:
I would prefer to avoiding using `lambda` if it is possible:
```python
self._queue = [item for item in self._queue if item[1].channel != channel]
```
I think `lambda` easily makes us do mistakes..